rancher-ui-2.10.yaml

```diff
@@ -0,0 +1,98 @@
+package:
+  name: rancher-ui-2.10
+  version: "2.10.4"
+  epoch: 0
+  description: Rancher UI
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - npm
+      - nvm
+      - perl
+      - py3-supported-build-base-dev
+      - python-3.10
+      - yarn
+  environment:
+    CI_BRANCH: ${{package.version}}
+    CI_BUILD_TAG: ${{package.version}}
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rancher/ui
+      tag: v${{package.version}}
+      expected-commit: b7670e7ae50a63e95feb0926ef2744d46dcab2a1
+      recurse-submodules: true
+
+  - runs: |
+      # Nothing newer version of nodejs is working with node-sass
+      source /usr/share/nvm/nvm.sh
+      nvm install v14
+      nvm use v14
```

Comment:
We shouldn't need to do this. We have a package for 14

Reply:
I don't think we have, no results:
```
wolfictl apk ls https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz | grep nodejs-14
```

I noticed we used `nvm install` in the other packages too.

Reply:
We had it at one point, but appear to have dropped it. It's still installable